pt/development/sessions.rst

```diff
@@ -1,17 +1,424 @@
+extensão ``session`` nativa do PHP. As sessões permitem identificar usuários 
+únicos em solicitações e armazenar dados persistentes para usuários específicos. 
+Ao contrário dos cookies, os dados da sessão não estão disponíveis no lado do cliente. 
+O uso de ``$_SESSION`` geralmente é evitado no CakePHP, e o uso das classes Session é preferido.
+
 .. _session-configuration:
 
-Session Configuration
+Configuração da Sessão
 =====================
 
+A configuração da sessão é geralmente definida em ``/config/app.php``. As opções disponíveis são:
+
+* ``Session.timeout`` - O número de *minutos* antes que o manipulador de sessões do CakePHP expire a sessão
+
+* ``Session.defaults`` - Permite usar as configurações de sessão padrão incorporadas como base para sua 
+  configuração de sessão. Veja abaixo os padrões internos.
+
+* ``Session.handler`` - Permite definir um manipulador de sessão personalizado. O banco de dados 
+  principal e os manipuladores de sessão de cache usam isso. Veja abaixo informações adicionais sobre manipuladores de sessão.
+
+* ``Session.ini`` - Permite definir configurações adicionais de sessão ini para sua configuração. Isso 
+  combinado com ``Session.handler`` substitui os recursos de manipulação de sessão personalizados das versões anteriores
+
+* ``Session.cookie`` - O nome do cookie em uso, o padrão é 'CAKEPHP'.
+
+* ``Session.cookiePath`` - O caminho da URL para o qual o cookie de sessão está definido. Mapeia para a configuração 
+  php.ini ``session.cookie_path``. O padrão é o caminho base do aplicativo.
+
+O padrão do CakePHP ``session.cookie_secure`` é ``true``, quando seu aplicativo está em um protocolo SSL. 
+Se seu aplicativo atender a partir de protocolos SSL e não SSL, você poderá ter problemas com a perda de sessões. 
+Se você precisar acessar a sessão nos domínios SSL e não SSL, desabilite isso::
+
+    Configure::write('Session', [
+        'defaults' => 'php',
+        'ini' => [
+            'session.cookie_secure' => false
+        ]
+    ]);
+
+O caminho do cookie da sessão é padronizado como o caminho base do aplicativo. Para mudar isso, 
+você pode usar o valor ini ``session.cookie_path``. Por exemplo, se você deseja que sua sessão 
+persista em todos os subdomínios, você pode::
+
+    Configure::write('Session', [
+        'defaults' => 'php',
+        'ini' => [
+            'session.cookie_path' => '/',
+            'session.cookie_domain' => '.yourdomain.com'
+        ]
+    ]);
+
+By default PHP sets the session cookie to expire as soon as the browser is
+closed, regardless of the configured ``Session.timeout`` value. The cookie
+timeout is controlled by the ``session.cookie_lifetime`` ini value and can be
+configured using::
+
+Por padrão, o PHP define o cookie da sessão para expirar assim que o navegador é fechado, 
+independentemente do valor configurado ``Session.timeout``. O tempo limite do cookie é 
+controlado pelo valor ini ``session.cookie_lifetime`` e pode ser configurado usando:
```

Comment:
Need a second : at the end of this line.

Reply:
Ops! I'ts not ready